discovery-provider/src/queries/attestation.py

```diff
@@ -0,0 +1,155 @@
+from typing import List, Tuple
```

Comment:
following convention from the dir, should this be get_attestation.py? or perhaps even a subdir in queries for this stuff

Reply:
yeah will change. I was originally thinking this was gonna be a POST so I didn't give it get, but it's a GET, so, get